### PR TITLE
Staging next gnome python

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by jtojnar** [comment]
Are import checks needed when there are tests? Those should already check that libraries can be imported.

File: pkgs/development/python-modules/statmake/default.nix [hunk 1]
```diff
@@ -24,6 +24,13 @@ buildPythonPackage rec {
     sha256 = "OXhoQAD4LEh80iRUZE2z8sCtWJDv/bSo0bwHbOOPVE0=";
   };
 
+  postPatch = ''
```
Comment (jtojnar) [inline pkgs/development/python-modules/statmake/default.nix:27]: Why move `postPatch` up? It is lower level action executed with other phases so so far we have been always moving it down (see also https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887).

Comment (SuperSandro2000) [reply pkgs/development/python-modules/statmake/default.nix:27]: Seeing that post for the first time. So far everything I have seen is ordered in the order how it is run, not some higher, middle lower level ordering which is very abstract. I can drop this again and we move this discussion to another time.

**Comment by SuperSandro2000** [comment]
> Are import checks needed when there are tests? Those should already check that libraries can be imported.

Yeah, we added them for simplicity and to check if checkInputs shadows some imports.

**Comment by SuperSandro2000** [comment]
@jtojnar I just saw your comment in the other thread and I am going to plug you in. I hope that is fine for you. I also found a bug in cffsubr which breaks the build which I am also to fix.

**Review by jonringer — APPROVED** [review]
LGTM

failures are broken on target branch
```
https://github.com/NixOS/nixpkgs/pull/157516

2 packages failed to build:
python310Packages.statmake python310Packages.ufo2ft

6 packages built:
python310Packages.cffsubr python310Packages.ufoLib2 python39Packages.cffsubr python39Packages.statmake python39Packages.ufo2ft python39Packages.ufoLib2
```

**Pull request merged**